scripts/linters/pylint_extensions_test.py

```diff
@@ -4021,111 +3976,110 @@ def test_no_space_after_keyword(self) -> None:
+        with tempfile.NamedTemporaryFile() as temp_file:
+            filename = temp_file.name
 
-        with utils.open_file(filename, 'w') as tmp:
-            tmp.write(
-                u"""
-                if  False:
-                    pass
-                elif  True:
-                    pass
-                while  True:
-                    pass
-                yield  1
-                return True if  True else False
-                """)
-        node_multiple_spaces_after_keyword.file = filename
-        node_multiple_spaces_after_keyword.path = filename
-
-        self.checker_test_object.checker.process_tokens(
-           pylint_utils.tokenize_module(node_multiple_spaces_after_keyword))
-
-        if_message = testutils.Message(
-            msg_id='single-space-after-keyword', args=('if'), line=2)
-        elif_message = testutils.Message(
-            msg_id='single-space-after-keyword', args=('elif'), line=4)
-        while_message = testutils.Message(
-            msg_id='single-space-after-keyword', args=('while'), line=6)
-        yield_message = testutils.Message(
-            msg_id='single-space-after-keyword', args=('yield'), line=8)
-        if_exp_message = testutils.Message(
-            msg_id='single-space-after-keyword', args=('if'), line=9)
-
-        with self.checker_test_object.assertAddsMessages(
-            if_message, elif_message, while_message, yield_message,
-            if_exp_message):
-            temp_file.close()
+            with utils.open_file(filename, 'w') as tmp:
+                tmp.write(
+                    u"""
+                    if  False:
+                        pass
+                    elif  True:
+                        pass
+                    while  True:
+                        pass
+                    yield  1
+                    return True if  True else False
+                    """)
+            node_multiple_spaces_after_keyword.file = filename
+            node_multiple_spaces_after_keyword.path = filename
+
+            self.checker_test_object.checker.process_tokens(
+               pylint_utils.tokenize_module(node_multiple_spaces_after_keyword))
+
+            if_message = testutils.Message(
+                msg_id='single-space-after-keyword', args=('if'), line=2)
+            elif_message = testutils.Message(
+                msg_id='single-space-after-keyword', args=('elif'), line=4)
+            while_message = testutils.Message(
+                msg_id='single-space-after-keyword', args=('while'), line=6)
+            yield_message = testutils.Message(
+                msg_id='single-space-after-keyword', args=('yield'), line=8)
+            if_exp_message = testutils.Message(
+                msg_id='single-space-after-keyword', args=('if'), line=9)
+
+            with self.checker_test_object.assertAddsMessages(
+                if_message, elif_message, while_message, yield_message,
+                if_exp_message):
+                pass
```

Comment:
ditto: indentation and concerns about `pass` in an assertion context manager. ditto for the other tests in this file